content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -26,18 +26,17 @@ You can use either `kubectl create configmap` or a ConfigMap generator in `kusto
 
 ### Create a ConfigMap Using kubectl create configmap
 
-Use the `kubectl create configmap` command to create configmaps from [directories](#create-configmaps-from-directories), [files](#create-configmaps-from-files), or [literal values](#create-configmaps-from-literal-values):
+Use the `kubectl create configmap` command to create ConfigMaps from [directories](#create-configmaps-from-directories), [files](#create-configmaps-from-files), or [literal values](#create-configmaps-from-literal-values):
 
 ```shell
 kubectl create configmap <map-name> <data-source>
 ```
 
 where \<map-name> is the name you want to assign to the ConfigMap and \<data-source> is the directory, file, or literal value to draw the data from.
 
-The data source corresponds to a key-value pair in the ConfigMap, where
+When you are creating a ConfigMap based on a file, the key in the \<data-source> will default to the basename of the file, and the value will default to the file content.
```

Comment:
nit:
```suggestion
When you are creating a ConfigMap based on a file, the key in the \<data-source> defaults to the basename of the file, and the value defaults to the file content.
```

Reply:
@sftim Totally agree. Somehow I keep making that mistake and I hope to get better.